packages/cli/core/src/commands/ui/deploy.spec.ts

```diff
@@ -465,6 +473,69 @@ describe('ui:deploy', () => {
         }
       );
 
+    test
+      .stdout()
+      .stderr()
+      .do(() => {
+        doMockConfirm();
+        mockedConfirm.mockResolvedValueOnce(true);
+        mockHostedPageList.mockResolvedValueOnce({
+          items: [{name: validJsonConfig.name, id: 'thisistheid'}],
+        });
+      })
+      .command(['ui:deploy'])
+      .it(
+        'should call hostedPages.list when no page id argument is passed and a page with the same already exists',
+        () => {
+          expect(mockHostedPageList).toHaveBeenCalledWith(
+            expect.objectContaining({
+              filter: validJsonConfig.name,
+              perPage: 1000,
+            })
+          );
+        }
+      );
+
+    test
+      .stdout()
+      .stderr()
+      .do(() => {
+        doMockConfirm();
+        mockedConfirm.mockResolvedValueOnce(true);
+        mockHostedPageList.mockResolvedValueOnce({
+          items: [{name: validJsonConfig.name, id: 'thisistheid'}],
+        });
+      })
+      .command(['ui:deploy'])
```

Comment:
This block seems the same as 476 - 485? If yes how are expecting different results than the previous? Are the checks broken up to have one `expect` per test?

Reply:
- [ ] group up the expect
- [ ] reword `it` to be clearer